libs/sdk-ui-ext/src/insightView/InsightView.tsx

```diff
@@ -240,6 +240,20 @@ class InsightViewCore extends React.Component<IInsightViewProps & WrappedCompone
         }
     };
 
+    private getBackendWithTelemetry = (): IAnalyticalBackend => {
+        const telemetryProps: object = {
+            ...this.props,
+        };
+
+        // add a fake prop so that the type of the visualization rendered is present in the telemetry
+        if (this.state.insight?.insight.visualizationUrl) {
```

Comment:
there is a function to get the visualization url..

Reply:
🤦  you are right, will fix asap